apps/system/js/pin_page_system_dialog.js

```diff
@@ -128,7 +128,6 @@
     switch (action) {
       case 'pin':
         activeApp.appChrome.pinPage();
-        this._banner.show();
```

Comment:
We should probably remove the _banner in the dialog, and use the systemBanner in the appChrome for all the cases